module3/lessons/Week2/FindingYourStrengths.md

```diff
@@ -0,0 +1,153 @@
+
+### Example #2
+Let’s look at another strength. A common strength that comes up for students is Persistence, which is defined as the drive to firmly continue in a course of action, despite difficulties, opposition, or warning; stubborn determination.
+
+* A **skill** that would contribute to this is detail-oriented planning. If you’re someone who uses a planner, likes to write to-do lists, and approaches projects from a backwards planning stance, this might describe you!
+
+* The **knowledge** needed here would be around knowing how to prioritize, which comes down to the ability to know which task to focus on at a time based on practice.
+
+* The natural **talent** that might come into play here is the ability to maintain focus during arduous tasks.
+
+Put together, these create the **strength** of persistence.
+
+</section>
+
+## Additional Strategies
+Breaking down strengths in this way can be easier when we already know what the strength is, but how can you approach identifying your strengths on your own?
+
+<section class='answer' markdown='1'>
+### 1. Use the SIGN method
+<section class='note' markdown='1'>
+* Success: Using your strengths makes you feel successful
+* Instinct: Activities you are naturally drawn to
+* Growth: Where you learn the most, come up with new ideas, and have the best insights
+* Needs: Where you feel the need to spend the most time
+</section>
+</section>
+
+
+<section class='answer' markdown='1'>
+### 2. Use others to find your strengths
+<section class='note' markdown='1'>
+* What do others ask for your help with?
+* What do others want you to teach them?
+* What do you receive compliments for?
+</section>
+</section>
+
+<section class='answer' markdown='1'>
+### 3. Use a Strenghs Assessment
+<section class='note' markdown='1'>
+* Use a tool like High 5, StrengthsFinder or Pairin as a starting point
+* Use your own experience to add a narrative to your identified strengths
+</section>
+</section>
+
+## Discussing Our Strengths
+
+<section class='call-to-action' markdown='1'>
+In small groups    
+
+Discuss the results of your High 5 assessment.  Use the following questions as a guide:
+* What strengths to do you most identify with?
+* Were your results suprising?
+* Are there any strengths that you do not really connect with? (this test is not perfect!)
+
+Together, help each other identify one additional strength.  You can use these resources for inspiration:
+* [High 5 Strenghts](https://high5test.com/wp-content/uploads/2018/04/HIGH5-Strengths-Sequence-Sample.pdf)
+* [Strengths to List on Your Resume](https://www.indeed.com/career-advice/resumes-cover-letters/strength-in-resume)
+* [Personal Strengths Defined](https://positivepsychology.com/what-are-your-strengths/)
+
+</section>
+
+<section class='call-to-action' markdown='1'>
+
+Individually  
+
+For one of your 6+ strengths identified previously, reflect on what **skill, knowledge, and talent** combine to make that **strength**.
```

Comment:
I think we want students to leave this session with roughly 3 strengths that they identified as important for them. I think keeping a list of 6+ will get too messy. 

How about we have them identify 3 of your main strengths, then break down the skill/knowledge/talent for one of those. And then they send that to us as a deliverable and starting point for the next sessions?

Reply:
Great idea - I agree that 6 is a lot to focus on.  I like the additional reflection to come up with those top 3, and it makes for an interesting deliverable :)